dvl-a50/mavlink2resthelper.py

```diff
@@ -166,24 +169,29 @@ def __init__(self):
 }}
 """
 
-    def get_float(self, path: str) -> float:
+    def get_float(self, path: str, vehicle: int=None, component: int=None) -> float:
         """
-        Helper to get mavlink data from mavlink2rest
+        Helper to get mavlink data from mavlink2rest.
+        Uses initialised vehicle and component as defaults, unless overridden.
         Example: get_float('/VFR_HUD')
-        Returns the data as a float or False on failure
+        Returns the data as a float (nan on failure)
         """
-        response = request(MAVLINK2REST_URL + '/mavlink' + path)
+        response = self.get(path, vehicle, component)
         if not response:
             return float("nan")
         return float(response)
 
-    def get(self, path: str) -> str:
+    def get(self, path: str, vehicle: int=None, component: int=None) -> str:
         """
         Helper to get mavlink data from mavlink2rest
+        Uses initialised vehicle and component as defaults, unless overridden.
         Example: get('/VFR_HUD')
         Returns the data as text or False on failure
         """
-        response = request(MAVLINK2REST_URL + '/mavlink' + path)
+        vehicle = vehicle or self.vehicle
+        component = component or self.component
+        vehicle_path = f"/vehicles/{vehicle}/components/{component}/messages"
+        response = request(MAVLINK2REST_URL + '/mavlink' + vehicle_path + path)
         if not response:
             return False
```

Comment:
Change if changing return type to `Optional[str]`:
```suggestion
            return None
```
Otherwise need to make the return type `Union[str, bool]`, or change the failure case to return an empty string. 

Returning `False`, `None`, or `""` will all still work fine in `if not response:` checks like the one in `get_float`.